onadata/apps/logger/migrations/0062_auto_20210202_0248.py

```diff
@@ -0,0 +1,25 @@
+# pylint: skip-file
+# Generated by Django 2.2.16 on 2021-02-02 07:48
+
+from django.db import migrations
+from onadata.apps.logger.models.instance import Instance
+
+
+def regenerate_instance_json(apps, schema_editor):
+    """
+    Regenerate Instance JSON
+    """
+    for inst in Instance.objects.filter(deleted_at__isnull=True):
+        inst.json = inst.get_full_dict(load_existing=False)
+        inst.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('logger', '0061_auto_20200713_0814'),
+    ]
+
+    operations = [
+        migrations.RunPython(regenerate_instance_json)
+    ]
```

Comment:
We will need a strategy to run this in our production environment, other deployments might be okay.